### ISSUE TITLE
[Bug]: Inertia scrolling paper when it's not supposed to

### ISSUE THREAD

**Issue by alexandernst** [clientIO/joint#2292]
### What happened?

I discovered that when I use a trackpad with [sensitivity & speed & fastclick] set to very high, Inertia will almost always trigger an unwanted `onScroll` event when I click a couple of times on a blank area (thus resulting in my paper moving around).

I have been debugging the issue and it seems that it's caused because the velocity isn't set to 0 when the panning is stopped (`handleDragEnd`).

I propose the following patch:

```diff
Inertia.prototype.handleDragEnd = function(_event) {
    this._isDragging = false;

+   this._velocityX = 0;
+   this._velocityY = 0;
}
```

### Version

3.7.1 (JJ+)

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari

### What operating system are you seeing the problem on?

Mac

**Comment by zbynekstara** [comment]
Hi @alexandernst, thank you for reporting this, but I have trouble reproducing this issue. Can you try and see if you can reproduce the issue in one of the built-in JointJS+ apps (you may need to set `paperScroller.inertia: true`)?

I see that you have a Mac, which should allow me to reproduce - but perhaps my trackpad settings are not matching yours. Can you share (screenshot?) your `Trackpad` and `Accessibility > Pointer Control` System Settings with me?

**Comment by alexandernst** [comment]
Hi! Of course!

Here is a demo: https://codesandbox.io/s/rappid-inertia-bug-5435r7?file=/src/index.js

These are my trackpad settings:

![image (1)](https://github.com/clientIO/joint/assets/89727/8497e6bc-b5d2-4cf0-9595-361d6150cc50)

And here is a repro video:

https://github.com/clientIO/joint/assets/89727/a1335312-8025-48c1-9341-7ee1cb00a161

Instructions:

Just touch the trackpad a few times while the cursor is inside the blank area (move it a little bit after each touch)
When I say "touch" I literally mean "touch", not "click". Don't make the trackpad trigger the haptic feedback.

**Comment by zbynekstara** [comment]
Thank you for the detailed instructions, I managed to reproduce the bug on my side! I am looking into the issue now.

**Comment by zbynekstara** [comment]
Thanks @alexandernst for the report and the reproducible example. The fix will be part of the JointJS+ patch 3.7.2 coming very soon (along with the fix for #2271).
